src/Dofs/DofHandler.jl

```diff
@@ -230,16 +227,10 @@ function _create_sparsity_pattern(dh::DofHandler, sym::Bool)
 end
 
 
-vtk_grid(filename::AbstractString, dh::DofHandler) = vtk_grid(filename, dh.grid)
+WriteVTK.vtk_grid(filename::AbstractString, dh::DofHandler) = vtk_grid(filename, dh.grid)
 
 # Exports the FE field `u` to `vtkfile`
-function vtk_grid(filename::AbstractString, dh::DofHandler, u::Vector)
```

Comment:
This felt wrong for `vtk_grid` so I removed it, instead do

```julia
vtkfile = vtk_grid("filename", dh)
vtk_point_data(vtkfile, dh, u)
```

which is a bit more transparent, ok? :p